src/components/Contentful/JourneyCardCarousel.vue

```diff
@@ -251,13 +259,13 @@ const orderedSlides = computed(() => {
 		];
 	}
 
-	if (props.slidesNumber) {
-		sortedSlides = sortedSlides.slice(0, props.slidesNumber);
+	if (shouldShowGoalCard.value) {
+		// Add empty slide at start for goal card
```

Comment:
Looks like GoalCard condition is based on an index `showGoalCard(index)`
Are we guaranteed always visible, or we could be adding this slide at start but not showing a Goal Card?